docs/DeveloperGuide.md

```diff
@@ -207,6 +207,31 @@ The following activity diagram summarizes what happens when a teacher executes a
     2. Only can set the class to a date at most 1 week away.
   
 
+### Schedule List Feature
+This feature allows the user to be able to view a schedule on the right hand side of the panel.
+
+#### Implementation Details
+In order to update the `UI` from original view of a single panel to the right panel, there was a
```

Comment:
I am personally not very sure about how we should write our implementation details for the front end and feel like less reasoning should be included here? So I am envisioning something like this for the current paragraph:

`MainWindow.fxml` is modified to update the `UI` from the original single-panel view to include an additional right panel.

What do you think? @cadencjk @gnahzeus @sjoann

Reply:
I agree with @Echomo-Xinyu, the sentence he gave is more concise but yet holds the same meaning.